tom_antares/antares.py

```diff
@@ -318,6 +329,7 @@ def fetch_alerts(self, parameters: dict) -> iter:
         mag_max = parameters.get('mag__max')
         elsquery = parameters.get('esquery')
         ztfid = parameters.get('ztfid')
+        max_alerts = parameters.get('max_alerts')
```

Comment:
should probably be `get('max_alerts', 20)` or something. This isn't backwards compatible with old queries or an empty `max_alerts` field.

Reply:
Yes, I agree.  A default maximum of 20 is reasonable, since its roughly what can be displayed on a single page.  And this allows a user to request more if they wish.